demo/components/http-endpoint/http-endpoint-service.js

```diff
@@ -34,9 +36,11 @@ app.service('HttpEndpoint', function($q, $http, $timeout) {
         }
       });
     },
-    load : function(resource, graphUri) {
+    load : function(resource, barEndpoint, graphUri) {
```

Comment:
What's up with the `foo` and `bar` prefixes? 😅

Reply:
lol.. have no idea, but kept the concept!!